overrides/scripts/AdvRocketry.zs

```diff
@@ -348,6 +354,161 @@ recipes.addShaped(<advancedrocketry:warpcore>, [
 	[<gregtech:compressed_3:10>, <extendedcrafting:storage:2>, <gregtech:compressed_3:10>],
 	[<thermalfoundation:storage_alloy:6>, <gregtech:compressed_3:10>, <thermalfoundation:storage_alloy:6>]]);
 
+// Rocket Monitoring Station
+recipes.remove(<advancedrocketry:monitoringstation>);
+recipes.addShaped(<advancedrocketry:monitoringstation>, [
+	[<gregtech:meta_item_1:14018>, <advancedrocketry:satelliteprimaryfunction>, <gregtech:meta_item_1:14018>],
+	[<gregtech:meta_item_1:14018>, <libvulpes:structuremachine>, <gregtech:meta_item_1:14018>],
+	[<gregtech:meta_item_1:14018>, <actuallyadditions:item_battery>, <gregtech:meta_item_1:14018>]
+]);
+
+// Guidance Computer Access Hatch
+recipes.remove(<advancedrocketry:loader:6>);
+recipes.addShaped(<advancedrocketry:loader:6>, [
+	[null, <ore:circuitAdvanced>, null],
+	[<gregtech:meta_item_1:14018>, <libvulpes:structuremachine>, <gregtech:meta_item_1:14018>],
+	[null, <ore:circuitAdvanced>, null]
+]);
+
+// Altitude Controller
+recipes.remove(<advancedrocketry:altitudecontroller>);
+recipes.addShapeless(<advancedrocketry:altitudecontroller>, [<libvulpes:structuremachine>,<ore:circuitBasic>,<extrautils2:screen>]);
+
+// Station Docking Port
+recipes.remove(<advancedrocketry:stationmarker>);
+recipes.addShapeless(<advancedrocketry:stationmarker>, [
+	<actuallyadditions:item_battery>, <ore:circuitGood>, <libvulpes:structuremachine>
+]);
+
+// Station Gravity Controller
+recipes.remove(<advancedrocketry:gravitycontroller>);
+recipes.addShapeless(<advancedrocketry:gravitycontroller>, [
+	<libvulpes:structuremachine>, <minecraft:piston>, <extrautils2:screen>
+]);
+
+// Altitude Controller
+recipes.remove(<advancedrocketry:altitudecontroller>);
+recipes.addShapeless(<advancedrocketry:altitudecontroller>, [
+	<libvulpes:structuremachine>, <extrautils2:screen>, <ore:circuitGood>
+]);
+
+// Orientation Controller
+recipes.remove(<advancedrocketry:orientationcontroller>);
+recipes.addShapeless(<advancedrocketry:orientationcontroller>, [
+	<libvulpes:structuremachine>, <extrautils2:screen>, <minecraft:compass>
+]);
+
+// Docking Pad
+recipes.remove(<advancedrocketry:landingpad>);
+recipes.addShapeless(<advancedrocketry:landingpad>, [
+	<advancedrocketry:launchpad>, <ore:circuitGood>
+]);

```

Comment:
There were two recipes added for the docking pad, I think the other one is better than this one